cmd/broker/deprovisioning_test.go

```diff
@@ -156,3 +162,56 @@ func TestDeprovisioning_HappyPathAWS(t *testing.T) {
 	suite.WaitForOperationsNotExists(iid)
 
 }
+
+func TestRuntimesEndpointForDeprovisionedInstance(t *testing.T) {
```

Comment:
this test shows the endpoint looks into instances_archived for deprovisioned instances